### DISCUSSION TITLE
Isn't the settings pop-up  too small?

### DISCUSSION THREAD

**Discussion by HubKing** [openstyles/stylus#1593] [General]
On my computer, it is too small that even the text on the drop-down menus are clipped with ellipses.  Also, this makes the existing style search list small (only 2 results are visible at a time).  Isn't it better that it is larger?

![image](https://user-images.githubusercontent.com/10900489/235369405-92dc52dc-3d3c-4e90-af8d-6caa29a423ad.png)

**Comment by tophf** [comment]
Normally it looks like this:

![image](https://user-images.githubusercontent.com/1310400/235370661-de2dcdab-3a3d-4c36-8f41-5049cddac591.png)

You can customize the width in the options. By default it's 246px. The height expands automatically up to 600px (the hardcoded maximum in Chrome/Firefox), so if it doesn't for you there may be a bug in the browser or OS that messes up the automatic resizing.

If you want to see more results at once you can open a new tab via the dropdown menu to the right of `Find styles` button.

**Reply by HubKing** [reply]
Well, I could set the width, but the font is too small. There is not an option to scale the pop-up or set the font size, is there? To be honest, other extensions' pop-ups are also small, and I think this is because FireFox on Linux does not apply Gnome's font scaling to pop-ups, and there is no zoom setting for extension pop-ups, and the ctrl+wheel zoom on the pop-ups is not remembered .

For size comparison, here is the relative size of the pop-up compared to other apps.
![image](https://user-images.githubusercontent.com/10900489/235373253-80e639ac-4baa-4a40-8892-2c54a75a94ed.png)

**Reply by tophf** [reply]
You can make a new style for the extension itself and adjust the font:
```css
@-moz-document url-prefix("chrome-extension://"), url-prefix("moz-extension://") {
  body {
    font-size: 20px;
  }
}
```

**Reply by HubKing** [reply]
> You can make a new style for the extension itself and adjust the font:

Thanks. But is it possible to apply the CSS only to toolbar pop-ups (like the one that appears when you click the [S] icon), not the extension tab page (like the "Edit style" page of Stylus)?

**Reply by tophf** [reply]
Yes. Replace `body` with `#stylus-popup`.
